src/Controller/CompanyController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
```

Comment:
we will use strict types:
declare(strict_types=1);
https://www.phptutorial.net/php-tutorial/php-strict_types/